src/raven.js

```diff
@@ -813,8 +813,15 @@ Raven.prototype = {
         // Capture breadcrubms from any click that is unhandled / bubbled up all the way
         // to the document. Do this before we instrument addEventListener.
         if (this._hasDocument) {
-            document.addEventListener('click', self._breadcrumbEventHandler('click'));
-            document.addEventListener('keypress', self._keypressEventHandler());
+            if (document.addEventListener) {
+                document.addEventListener('click', self._breadcrumbEventHandler('click'));
+                document.addEventListener('keypress', self._keypressEventHandler());
+            }
+            else {
+                // IE8 Compatibility
+                document.attachEvent('onclick', self._breadcrumbEventHandler('click'));
```

Comment:
FWIW I considered the case where we somehow got here but aren't in IE. Perhaps we should first check for `attachEvent`? And I'm not sure what we should do if we don't have either...